app/vue/src/api.ts

```diff
@@ -1,4 +1,8 @@
-const URL = process.env.VUE_APP_API_URL
+export const URL = process.env.VUE_APP_API_URL
+
+export const ACCESS_TOKEN = process.env.VUE_APP_ACCESS_TOKEN
+
+export const nominatimURL = process.env.VUE_APP_NOMINATIM_URL
```

Comment:
How important is pep 8? I see different naming styles here. Maybe we should use a style guide checker?
```suggestion
export const ACCESS_TOKEN = process.env.VUE_APP_ACCESS_TOKEN

export const NOMINATIM_URL = process.env.VUE_APP_NOMINATIM_URL
```

Reply:
Now I see it is front-end code. Typescript. So no PEP 8. But maybe the google coding style.

Reply:
Yes, this is good to keep consistent. I will update it.